launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/servlets/FormHandlerServlet.java

```diff
@@ -73,19 +72,23 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws Illeg
     datastore.put(userFeedEntity);
 
     // return accessible link to user
-    String urlID = KeyFactory.keyToString(userFeedEntity.getKey());
+    String urlID = KeyFactory.keyToString(userFeedEntity.getKey()); // the key string associated with the entity, not the numeric ID.
     String rssLink = BASE_URL + urlID;
     res.setContentType("text/html");
-    res.getWriter().println(rssLink);
+    res.getWriter().print(rssLink);
   }
 
   /**
-   * Display RSS feed xml string that user tries recalling with the given ID
+   * Display RSS feed xml string that user tries recalling with the given ID.
+   * @throws IOException
```

Comment:
nit: add doc for when exception is thrown. in this case "@throws IOException if no matching id is found in datastore"